tests/e2e/browser/設定ダイアログ/書き出しファイル名パターン.spec.ts

```diff
@@ -9,7 +9,10 @@ test.beforeEach(gotoHome);
  * 書き出しファイル名パターンダイアログまで移動
  */
 const moveToFilenameDialog = async (page: Page, settingDialog: Locator) => {
-  await settingDialog.getByRole("button", { name: "編集する" }).click();
+  const fileNamePatternCell = settingDialog.locator("div.q-card__actions", {
+    has: settingDialog.getByText("書き出しファイル名パターン"),
+  });
+  await fileNamePatternCell.getByRole("button", { name: "編集する" }).click();
```

Comment:
なぜこのコードがダメなのか分かりませんが、↓で行きました。。。
```suggestion
  await settingDialog
    .locator("div.q-card__actions", { hasText: "書き出しファイル名パターン" })
    .getByRole("button", { name: "編集する" })
    .click();
```

流石に`q-card__actions`クラスを前提にするのはなんか気が引けるので、こっちの方がいいかも。
```suggestion
  await settingDialog
    .locator("div", { hasText: "書き出しファイル名パターン" })
    .getByRole("button", { name: "編集する" })
    .click();
```

Reply:
ちょっとテストが動くか試しておいたほうがお互い楽な気がしたので、後者の方をコミットさせていただきます！
全然変えちゃっていただければ！！

Reply:
あっテスト落ちちゃった。。。。。。`div.q-card__actions`にしないとすごい広い範囲のdivも取れてしまってるのかも。。。すみません。。。。。。